Move prover returns: exiting with verification errors
error: post-condition does not hold
   ┌─ tests/sources/functional/invariants_resources.move:31:9
   │
31 │         ensures result < 1;
   │         ^^^^^^^^^^^^^^^^^^^
   │
   =     at tests/sources/functional/invariants_resources.move:27: get_invalid
   =         a = <redacted>
   =     at tests/sources/functional/invariants_resources.move:28: get_invalid
   =         result = <redacted>
   =     at tests/sources/functional/invariants_resources.move:29: get_invalid
   =     at tests/sources/functional/invariants_resources.move:31: get_invalid (spec)
